### PR TITLE
Test music

### PR DESCRIPTION
**Pull request by henryzbw**
TT wants to click logo 18 times to play music

### CODE REVIEW

File: src/components/NavBar/components/ImageLocaleWrapper/index.js [hunk 1]
```diff
@@ -25,12 +25,14 @@ export default class ImageLocaleWrapper extends Component {
   }
 
   render() {
-    const { intl, src, alt, className } = this.props;
+    const { intl, src, alt, className, onClick } = this.props;
     return (
+      // eslint-disable-next-line
```
Comment (dwalintukan) [inline src/components/NavBar/components/ImageLocaleWrapper/index.js]: what needs to be disabled?

File: src/components/NavBar/index.js [hunk 2]
```diff
@@ -43,7 +56,11 @@ export default class NavBar extends Component {
         <Collapse in={!ui.searchBarMode}>
           <Toolbar className={classes.navBarWrapper}>
             <div className={classes.navSection}>
-              <BodhiLogo {...this.props} />
+              <BodhiLogo {...this.props} onClick={() => this.handleClickLogo()} />
+              {
+                // eslint-disable-next-line
```
Comment (dwalintukan) [inline src/components/NavBar/index.js]: is the audio class not allowed?

File: src/scenes/App/index.js [hunk 3]
```diff
@@ -28,6 +29,7 @@ const App = injectIntl(observer(({ classes, intl, match: { url }, store, store:
     {!store.loading && (
       <Fragment>
         <div className={classes.container}>
+          <NavBar />
```
Comment (dwalintukan) [inline src/scenes/App/index.js]: no props need to be passed to it?

File: src/components/NavBar/components/ImageLocaleWrapper/index.js (hunk 1 above)
Comment (henryzbw) [reply src/components/NavBar/components/ImageLocaleWrapper/index.js]: will have this warning, Non-interactive elements should not be assigned mouse or keyboard event listeners

File: src/components/NavBar/index.js (hunk 2 above)
Comment (henryzbw) [reply src/components/NavBar/index.js]: will have this error: Media elements such as <audio> and <video> must have a <track> for captions

File: src/scenes/App/index.js (hunk 3 above)
Comment (henryzbw) [reply src/scenes/App/index.js]: don't think we need any props. all the page route is working

**Pull request merged**